// *****************************************************************************
//
// This file is part of a MASA library or program.
// Refer to the included end-user license agreement for restrictions.
//
// Copyright (c) 2010 MASA Group
//
// *****************************************************************************

package sword;

message AarInformation {
    required string information = 1;
}

message PlotResult {
    required uint32 identifier = 1;
    repeated float  values     = 2;
    required string error      = 3;
}

message Indicator {
    required string name  = 1;
    required float  value = 2;
}

message AarToClient {
    message Content {
        optional AarInformation aar_information = 1;
        optional PlotResult     plot_result     = 2;
        optional Indicator      indicator       = 3;
    }
    optional int32   context = 1;
    optional Content message = 2;
}
